### ISSUE TITLE
Contradictory outcomes

### ISSUE THREAD

**Issue by EduardoFG11** [vaibhavk97/GoBooDo#71]
Hi, complete utter noob here arriving to this program from a closed thread in reddit. I have had the following issues.

1. "name='NID', domain=None, path=None" Received invalid response
   -  I then turned on my own VPN and connected from the US and tried again.
   
2. "Please delete the corresponding folder and start again or the book is not available for preview.
Received invalid response"
  - I then tried with another book
  
3. "Downloading Index des livres interdits - JesÃºs MartÃ­nez de Bujanda Received invalid response"

Would love any ELI5 help and sorry for my ignorance of all things Python/code!

**Comment by impulsen52** [comment]
> Hi, complete utter noob here arriving to this program from a closed thread in reddit. I have had the following issues.
> 
> 1. "name='NID', domain=None, path=None" Received invalid response
>    
>    * I then turned on my own VPN and connected from the US and tried again.
> 2. "Please delete the corresponding folder and start again or the book is not available for preview.
>    Received invalid response"
> 
> * I then tried with another book
> 
> 3. "Downloading Index des livres interdits - JesÃºs MartÃ­nez de Bujanda Received invalid response"
> 
> Would love any ELI5 help and sorry for my ignorance of all things Python/code!

You need to install html5lib
`pip install html5lib --user`
